api/users/serializers.py

```diff
@@ -128,6 +134,44 @@ def profile_image_url(self, user):
         size = self.context['request'].query_params.get('profile_image_size')
         return user.profile_image_url(size=size)
 
+    def _update_social(self, social_dict, user):
```

Comment:
This was pulled from the first part of the `update` method? doesn't look like it's being used.